source/crud/tests/v2/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,35 @@
+runOn:
+    -
+        # arrayFilters support first introduced in 3.5.6
+        maxServerVersion: "3.5.5"
+
+data:
+    - {_id: 1, y: [{b: 3}, {b: 1}]}
+    - {_id: 2, y: [{b: 0}, {b: 1}]}
+
+collection_name: &collection_name "test"
+database_name: &database_name "crud-tests"
+
+tests:
+  -
+    description: "BulkWrite on server (that doesn't support arrayFilters) with arrayFilters on second op"
```

Comment:
I realize it's outside of the scope of [SPEC-1691](https://jira.mongodb.org/browse/SPEC-1691), but I don't believe we have an existing test for raising an error if arrayFilters is used in the first op. Would you mind adding another test case for that while you're creating this file?

Reply:
Done.